pkg/controllers/subnet/subnet_controller.go

```diff
@@ -76,7 +85,11 @@ func (r *SubnetReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctr
 				obj.Spec.IPv4SubnetSize = vpcNetworkConfig.DefaultIPv4SubnetSize
 			}
 		}
-		if _, err := r.Service.CreateOrUpdateSubnet(obj, nil); err != nil {
+		var tags []model.Tag
+		for k, v := range nsObj.Labels {
```

Comment:
in NCP we normalize the NS labels, since NSX tags have length limit.

Reply:
Yes, aligned with yun, we would submit a new patch to refactor the tags of CRs.